maybe/syscall_filters.py

```diff
@@ -108,6 +109,30 @@ def substitute_open(path, flags):
         return None
 
 
+def format_mknod(path, type):
+    path = abspath(path)
+    if path in allowed_files:
+        return None
+    elif (type & S_IFREG) and not exists(path):
+        return "%s %s" % (T.cyan("create file"), T.underline(path))
+    elif (type & S_IFIFO) and not exists(path):
+        return "%s %s" % (T.cyan("create named pipe"), T.underline(path))
+    else:
+        # TODO: add support for block and char special files
+        return None
+
+
+def format_mkfifo(path):
```

Comment:
Instead of having a separate method, call `format_mknod` with `S_IFIFO` :sunglasses: